files/build_templates/sonic_debian_extension.j2

```diff
@@ -169,7 +166,6 @@ sudo cp -f $IMAGE_CONFIGS/systemd/journald.conf $FILESYSTEM_ROOT/etc/systemd/
 
 # Copy interfaces configuration files and templates
 sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config.service  $FILESYSTEM_ROOT/etc/systemd/system/
-sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable interfaces-config.service
```

Comment:
keep

Reply:
This will need to be moved to the generator for multi-ASIC functionality